pkg/ccl/importccl/read_import_mysql.go

```diff
@@ -306,6 +333,16 @@ func mysqlTableToCockroach(
 		if err != nil {
 			return nil, nil, err
 		}
+		if raw.Type.Autoincrement {
+			if seqName == "" {
+				return nil, nil, errors.Errorf("column %q specifies AUTO_INCREMENT but table options did not include it", def.Name)
+			}
+			expr, err := parser.ParseExpr(fmt.Sprintf("nextval('%s':::STRING)", seqName))
```

Comment:
is it possible for seqName to contain a single quote? Seems possible if the table name contains a single quote?

Reply:
Added another commit that uses [`CompliantName`](https://github.com/vitessio/vitess/blob/master/go/vt/sqlparser/ast.go#L3597) everywhere, which replaces any non-letter, and any non-digit or leading digit with `_` as far as I can tell.